docs/index.md

```diff
@@ -1,5 +1,11 @@
 # Jupyter Accessibility
 
+## Accessibility docs
+
+Looking for [accessibility docs?](accessibility-docs)
+
```

Comment:
Maybe this should be an admonition of some kind but I'm not super familiar with MyST and I didn't want to spend too much time on this

Reply:
Do we need to duplicate this one?

Reply:
I think so yes because the goal for me with this PR is if somebody lands on one of our jupyter-accessibility pages after doing a search for Jupyter(Lab) accessibility - then I want to point them to the page I created in this PR. 

The `README.md` file is shown in the front page of the GitHub repo, which is a high match in search engines for "jupyter" and "accessibility" - it's what you see at this URL: https://github.com/jupyter/accessibility

The `index.md` file is the front page for the team compass site, at this URL: https://jupyter-accessibility.readthedocs.io/en/latest/

I could potentially remove the pointer from `useful-resources.md`, but I think in that case we would need to restructure the organization of the team compass site. Because my concern with the current structure is that if someone lands on that page looking for VSCode-style user-facing accessibility docs, and they are trying to quickly locate something, their eye might land on "resources" and think that that's a match for what they're looking for, only to discover that it isn't. Because in this PR, I removed the link to the toolbar, the resources page after this PR only has resources that are mostly aimed at contributors or auditors. 

Or are you asking something entirely different? Are you asking if there's a way to include a markdown snippet across several files, rather than copy-pasted the same content? That I don't know.

Reply:
if i want to tab to the docs link, i'll have to tab through the header which seems like an unnecessary step.

i don't think that this should be heading. the call to action makes the most semantic sense as a link. an emoji like :bell:  would redirect peoples attention

Reply:
Thanks @tonyfast looking out for this 😌. I changed it as you suggested